compiler/vmdeps.nim

```diff
@@ -17,6 +17,7 @@ proc opSlurp*(file: string, info: TLineInfo, module: PSym; conf: ConfigRef): str
     result = readFile(filename)
     # we produce a fake include statement for every slurped filename, so that
     # the module dependencies are accurate:
+    conf.m.extraFileDeps.add filename
```

Comment:
Why not just ``discard conf.fileInfoIdx(filename)`` ?

Reply:
Then all the other changes you did are not required, maybe the rename from "nimfiles" to "depfiles" is a good one though.

Reply:
isn't fileInfoIdx reserved for nim (or nims) files? if not, I'll do as u suggest

Reply:
It should be fine and it will fix the incomplete IC implementation too.

Reply:
done, PTAL